### PR TITLE
adding the translator experimental with Keras

### PR DESCRIPTION
**Pull request by manimotion**
# Changes 🤖

- Added the File with the translator experimental with Keras
- Added the libraries in the Requirements file.

# Modified Data folders?  📚
- [ ] external
- [ ] interim
- [ ] processed
- [ ] raw

# Created new notebooks? 🐉
- [x] Yes
- [ ] No

# Created new reports? 📜
- [ ] Yes
- [x] No

# Created new models? 🐶
- [x] Yes
- [ ] No

# Do you feel happy with the changes? 🦥
- [x] Yes
- [ ] No

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,4 +14,6 @@ conda install -n xalaxians_env ipykernel --update-deps --force-reinstall
 conda install -c conda-forge transformers
 conda install pytorch
 conda install -c conda-forge pypdf2
-conda install -c anaconda scipy
+conda install -c anaconda scipy}
```
Comment (crisywini) [inline requirements.txt]: fix this please

**Review by crisywini — CHANGES_REQUESTED** [review]
Check the requirements

File: requirements.txt (hunk 1 above)
Comment (manimotion) [reply requirements.txt]: Fixed!

**Review by crisywini — APPROVED** [review]
Perfect! just update your changes with develop

**Pull request merged**